backend/src/main/java/com/mynt/banking/currency_cloud/manage/transactions/TransactionController.java

```diff
@@ -0,0 +1,31 @@
+package com.mynt.banking.currency_cloud.manage.transactions;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.mynt.banking.currency_cloud.manage.balances.requests.FindBalanceAllCurrencies;
+import com.mynt.banking.currency_cloud.manage.transactions.requests.FindTransaction;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+@RestController
+@RequestMapping("/api/v1/currency-cloud/transactions")
+@RequiredArgsConstructor
+public class TransactionController {
+
+    @Autowired
```

Comment:
I don't think autowired is recommended. Config and args constructors annotations can be used